pkg/control/v2/server/server.go

```diff
@@ -85,6 +85,7 @@ func (s *Server) Start() error {
 		s.logger.Errorf("unable to create listener: %s", err)
 		return err
 	}
+	s.logger.With("address", control.Address()).Infof("GRPC control socket listening at %s", control.Address())
```

Comment:
Doesn't this log the control address twice? Once in the message and once with the `address` key?

Reply:
Yes it does. I really want it to have the address key in the log line, but also having it in the message is nice. I don't think it hurts to have it twice.